es/sources_index_test.go

```diff
@@ -38,6 +40,15 @@ func (suite *SourcesIndexerSuite) TestSourcesIndex() {
 
 	fmt.Printf("\n\n\nAdding source.\n\n")
 	source1UID, source1ID := suite.us(Source{Name: "test-name-1"}, consts.LANG_ENGLISH)
+	fileHe := mdbmodels.File{ID: 1, Name: "heb_o_rav_bs-shamati-038-irat-ashem_2016-09-16_lesson.doc", UID: "8awHBfjU", Language: null.String{consts.LANG_HEBREW, true}, Secure: 0, Published: true}
+	fileEn := mdbmodels.File{ID: 2, Name: "eng_t_rav_2017-06-09_kitei-makor_mi-ichud-le-hafatza_n1_p1.docx", UID: "frnHYhIw", Language: null.String{consts.LANG_ENGLISH, true}, Secure: 0, Published: true}
+	fContentHe, err := es.DocText(fileHe.UID)
+	r.Nil(err)
+	fContentEn, err := es.DocText(fileEn.UID)
```

Comment:
Try to debug it and you will see that fContentEn is nil. Before calling DocText you should set the mock value.
e.g. 
suite.serverResponses["/doc2text/frnHYhIw"] = sourceContent